controllers/Sources.php

```diff
@@ -209,7 +209,7 @@ public function write() {
 
         $return = [
             'success' => true,
-            'id' => $id,
+            'id' => intval($id),
```

Comment:
This should be `(int) $id` cast for consistency (will be enforced once refactoring PR is merged).